scripts/bert/finetune_classifier.py

```diff
@@ -139,29 +142,30 @@
     'Options include \'book_corpus_wiki_en_cased\', \'book_corpus_wiki_en_uncased\''
     'for both bert_24_1024_16 and bert_12_768_12.'
     '\'wiki_cn_cased\', \'wiki_multilingual_uncased\' and \'wiki_multilingual_cased\''
-    'for bert_12_768_12 only.'
-)
+    'for bert_12_768_12 only.')
 parser.add_argument(
     '--pretrained_bert_parameters',
     type=str,
     default=None,
-    help='Pre-trained bert model parameter file. default is None'
-)
+    help='Pre-trained bert model parameter file. default is None')
 parser.add_argument(
     '--model_parameters',
     type=str,
     default=None,
     help='A parameter file for the model that is loaded into the model'
     ' before training/inference. It is different from the parameter'
-    ' file written after the model is trained. default is None'
-)
+    ' file written after the model is trained. default is None')
 parser.add_argument(
     '--output_dir',
     type=str,
     default='./output_dir',
     help='The output directory where the model params will be written.'
-    ' default is ./output_dir'
-)
+    ' default is ./output_dir')
+parser.add_argument(
+    '--only_inference',
+    action='store_true',
+    help='whether to do inference only on dev data. '
+    'If true, will load params from --model_parameters.')
```

Comment:
Can you add a test in `gluon-nlp/scripts/tests/test_bert.py`?

Reply:
Not sure how to do that. It needs a params file to do inference.

Reply:
Ok. Let's revisit it later once we have some pre-trained model uploaded that can be used for inference